django_netjsonconfig/base/admin.py

```diff
@@ -90,6 +91,9 @@ def get_urls(self):
             url(r'^preview/$',
                 self.admin_site.admin_view(self.preview_view),
                 name='{0}_preview'.format(url_prefix)),
+            url(r'^context/(?P<pk>[^/]+)/$',
+                self.device_context_view,
```

Comment:
you have to use `self.admin_site.admin_view` as in the other URLs (for some reason the schema view does not do that, but in this case we need it)